internal/release/promote/perform.go

```diff
@@ -107,45 +192,159 @@ func (p *Promotion) perform(params PerformParams) (string, error) {
 	return prURL, nil
 }
 
-// getPromotionMessage computes the message for a specific release promotion
-func getPromotionMessage(releaseName string, sourceRelease, targetRelease *v1alpha1.Release) string {
-	previousVersion := "(missing)"
-	if targetRelease != nil {
-		previousVersion = targetRelease.Spec.Version
+func getReviewers(info *PromotionInfo) []string {
+	uniqueAuthors := make(map[string]string)
+	for _, release := range info.Releases {
+		for _, commit := range release.Commits {
+			uniqueAuthors[commit.GitHubAuthor] = commit.GitHubAuthor
+		}
 	}
-	return fmt.Sprintf("Promote %s %s -> %s", releaseName, previousVersion, sourceRelease.Spec.Version)
+
+	var reviewers []string
+	for _, reviewer := range uniqueAuthors {
+		reviewers = append(reviewers, reviewer)
+	}
+	sort.Strings(reviewers)
```

Comment:
Not that it matters, but I son't think we need to sort it.

Reply:
The order would be random. I prefer to have consistent, predictable and idempotent results, which is essential for unit/integration tests, but also makes for a better UX that can be explained.

Reply:
Fair enough, it's not like this will be overly expensive. Even more so compared to fetching the authors in the first place. ;)